flowey/flowey_core/src/github_context.rs

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+//! Core types and traits used to read GitHub context variables.
+
+use crate::node::{user_facing::GhContextVar, ClaimVar, NodeCtx, ReadVar, StepCtx};
```

Comment:
each import on its own line please.

the guide has some suggestions on how to configure rust-analyzer to do this for you